go/vt/schemadiff/table.go

```diff
@@ -1631,7 +1631,15 @@ func (c *CreateTableEntity) apply(diff *AlterTableEntityDiff) error {
 			// we expect the named key to be found
 			found := false
 			switch opt.Type {
-			case sqlparser.NormalKeyType, sqlparser.PrimaryKeyType:
+			case sqlparser.PrimaryKeyType:
```

Comment:
This should be handled separately since we want to check the type in this case, not the name. There's only one primary key anyway so we need to find that.